diff --git a/dist/chunks/mermaid.core/chunk-HRU6DDCH.mjs b/dist/chunks/mermaid.core/chunk-HRU6DDCH.mjs
index e4a3916a6213361fbdff684bbdb50bf5a20a6916..a995a2be58911a5222dc1f73346d25345961021e 100644
--- a/dist/chunks/mermaid.core/chunk-HRU6DDCH.mjs
+++ b/dist/chunks/mermaid.core/chunk-HRU6DDCH.mjs
@@ -436,7 +436,8 @@ var rect = /* @__PURE__ */ __name(async (parent, node) => {
   const text2 = await createText(labelEl, node.label, {
     style: node.labelStyle,
     useHtmlLabels,
-    isNode: true
+    isNode: true,
+    width: 1000,
   });
   let bbox = text2.getBBox();
   if (evaluate(siteConfig.flowchart.htmlLabels)) {
